### DISCUSSION TITLE
How to determine and modify the Window's Size and Position?

### DISCUSSION THREAD

**Discussion by nghtstr** [fyne-io/fyne#5224] [Q&A]
I just got started using Fyne, and I am impressed with what I can do with it. That said, I am not sure if I am seeing things correctly.

I know that I can `Resize()` a window, that is not a problem. However, what I cannot seemingly do is set its current position. Also, I don't seem to have access as a getter to the current size or position of said window. My reasoning, before this gets asked, is that I would like to save using the Preferences API the current size and position of the window, so when the user relaunches the application, the window returns to the size and position it was last in. I was going to implement it in the `myWindow.SetCloseIntercept(func)` method.

Any help on this matter would be most appreciated.

**Comment by andydotxyz** [comment]
You are right these properties are not accessible with the public API - in general it leads to bad UX patterns. However we do agree that remembering positions would be good. Some OS do this already and others do not. For this reason we have a ticket to track that being added in Fyne so you never have to worry about adding it #554 .

**Comment by andydotxyz** [comment]
I guess that makes this a duplicate issue to the one l linked, so I will close this one.

**Reply by nghtstr** [reply]
While I agree that this could be considered a duplicate, I wasn't able to find the issue that you linked to when I was initially searching for this issue. Even if this was mentioned somewhere in the documentation (that saving the location of the window is not currently supported) would be a huge help.

**Reply by andydotxyz** [reply]
I agree with your point but we don't generally document what is not completed - instead using this issue tracker for that purpose.
Hopefully we can get that added in the near future and it's no longer a problem.
